### PR TITLE
New Country Singapore

### PR DESCRIPTION
**Pull request by ashlrem**
Added Fixed Holidays Implemented except for Vesak Day and Deepavali.

### CODE REVIEW

**Comment by gerryster** [comment]
@ashlrem: thanks for contributing! Singapore is on my list of countries to add so thanks for getting this started. A couple of pieces of initial feedback:

1. Make sure the unit tests are passing. You will need to create test fixtures for this country for 2017 - 2020. You can find examples of these JSON files in the test/fixtures directory. These are basically static definitions of the holidays that are created from your definitions. Check out CONTRIBUTING.md for more information on running the tests and creating these fixtures.
2. You can delete data/countries/SG.todo.
3. Assign this PR to @commenthol .

File: data/countries/SG.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    langs: []
+    zones:
+      - Asia/Singapore
+    dayoff: sunday
+    days:
+      01-01:
+        _name: 01-01
+        type: public
+      chinese 01-0-01:
+        type: observance
```
[the remark this answers is not in the document]
Comment (ashlrem) [reply data/countries/SG.yaml]: Hi gerryster, thank you for the feedback. I will make an update today regarding this. Apologies if my work is incomplete. Will commit an update later today. Thank you!

[the remark this answers is not in the document]
Comment (gerryster) [reply data/countries/SG.yaml]: No need for apologies. Thanks again for starting this!

I am honestly a bit confused about the schedule for Vesak (https://en.wikipedia.org/wiki/Vesak). I think that starting with the fixed date holidays makes a lot of sense.

**Comment by commenthol** [comment]
@ashlrem can you please review the changes made upon this PR in #77.

**Comment by commenthol** [comment]
Closing in favor of #77

**Pull request closed without merging**